rasa_sdk/interfaces.py

```diff
@@ -230,14 +231,17 @@ def name(self) -> Text:
         raise NotImplementedError("An action must implement a name")
 
     async def run(
-        self, dispatcher, tracker: Tracker, domain: Dict[Text, Any]
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
     ) -> List[Dict[Text, Any]]:
         """Execute the side effects of this action.
 
         Args:
             dispatcher: the dispatcher which is used to
                 send messages back to the user. Use
-                ``dipatcher.utter_message()`` or any other
+                ``dispatcher.utter_message()`` or any other
                 ``rasa_sdk.executor.CollectingDispatcher``
                 method.
```

Comment:
Since the other methods are now deprecated, would you mind updating this to say that the dispatcher sends messages using the `dispatcher.utter_message()` method?

Reply:
Sure! I will update this later and fix the tests

Reply:
Great! Am actually wondering why the tests are breaking, this doesn't seem like it should break anything. Perhaps it's an import loop...

Reply:
Yeah, that seems right. This is the output i get after running the tests locally 

![image](https://user-images.githubusercontent.com/2881357/80779563-63ff4e80-8b42-11ea-8207-36880132e45c.png)